compiler/common-artifacts/exclude.lst

```diff
@@ -178,3 +178,4 @@ tcgenerate(CircleFullyConnected_U4_002)
 tcgenerate(GRU_000) # luci-interpreter does not support custom GRU
 tcgenerate(InstanceNorm_000)
 tcgenerate(InstanceNorm_001)
+tcgenerate(RmsNorm_000)
```

Comment:
as this is exclude list, if `luci-interpreter` supports RmsNorm, plz remove this line.

Reply:
Okay, there is no error after removing that line. :)